app/services/order-service.js

```diff
@@ -118,12 +118,34 @@ export default ApiService.extend({
    * @return {Order} the editable draft order or appointment
    */
   getLastDraft({ appointment }) {
+    return this.getAllOnlineOrAppointmentOrder({ appointment: false }).then(
+      orders => {
+        return orders
+          .filterBy("state", "draft")
+          .sortBy("createdAt:desc")
+          .get("firstObject");
+      }
+    );
+  },
+
+  getAllOnlineOrAppointmentOrder({ appointment }) {
```

Comment:
I think the name is a bit confusing, maybe something simply having `fetchOnlineOrders()` is enough

Reply:
or `fetchOrders({ type: 'appointment' })`

Reply:
`OR` kinda implies that it will return both, which makes it confusing